doc/source/reference/release_2.0.rst

```diff
@@ -106,3 +106,10 @@ API changes
    Most of the shortest_path algorithms now raise a NodeNotFound exception
    when a source or a target are not present in the graph.
 
+* [`#2326 <https://github.com/networkx/networkx/pull/2326>`_]
+   Centrality algorithms were harmonized with respect to the defaul behavior of
+   the weight parameter. The default behavior is now to use `'weight'` as weight
+   keyword. This affects `betweenness_centrality()`, `edge_betweenness_centrality()`,
```

Comment:
Perhaps this would be easier to read as an alphabetized list? Also, these should be references to the actual functions:
```rest
This affects the following centrality functions.

- :func:`betweenness_centrality`
- :func:`edge_betweenness_centrality`
- ...
```